### PR TITLE
Fix cppcheck warnings

### PR DESCRIPTION
**Pull request by crbrox**
Related with https://github.com/telefonicaid/fiware-orion/issues/1368

### CODE REVIEW

File: src/lib/rest/httpRequestSend.cpp [hunk 1]
```diff
@@ -494,344 +494,5 @@ std::string httpRequestSend
   return resp;
 }
 
-#else // Old functionality()
```
Comment (fgalan) [inline src/lib/rest/httpRequestSend.cpp]: Please, delete also (many lines above in the code):

```
 //#define USE_OLD_SENDHTTPSOCKET
 #ifndef USE_OLD_SENDHTTPSOCKET
```

and

```
 #endif
```

**Comment by kzangeli** [comment]
LGTM

File: src/lib/rest/httpRequestSend.cpp (hunk 1 above)
Comment (crbrox) [inline src/lib/rest/httpRequestSend.cpp]: Fixed 737f8f5

**Comment by fgalan** [comment]
LGTM

**Pull request merged**